tools/schema/action.json

```diff
@@ -76,19 +76,26 @@
                                 "description": "Player's response",
                                 "type": "string"
                             },
-                            "delay_actions_by_ticks": {
+                            "hover_text": {
```

Comment:
Isn't hover_text an action? Does this go here? And if so, shouldn't it be a $ref to the definition below rather than a string?

Reply:
Scratch that, I missed that this whole thing is "action". Should still be a $ref, though, I think.

Reply:
I think this is ok as-is, it has a slightly different description for the two different places it's used.